config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests/:id", to: "guests#show"
+  
+  post "/guest_rooms", to: "guest_rooms#add_room"
```

Comment:
A lot of these routes are great. The only one that isn't quite correct is the one on line 9. The verb and controller are correct, but the path and action aren't. Remember that we want to stick to the standard actions.

This route should be: `post "/guest/:id/rooms", to: "guest_rooms#create"`

Whenever we are using POST, this is going to be a create action.